.circleci/config.yml

```diff
@@ -50,9 +51,9 @@ jobs:
     environment:
       CC_TEST_REPORTER_ID: 586f7041e9f71c44946cdfe8bce27d66c6271d173ec291c2a9aa7e1b56667873
     docker:
-      - image: ssmarco/php-solr:latest
+      - image: ssmarco/php-solr:7.4
```

Comment:
Hi @Firesphere I got new docker image with PHP 7.4 and Solr 8.11.1 in it.
CircleCI is still failing though so will have a look at it soon.

Reply:
Probably related to MySQL? I've had that a few times. Might be a simple case of environment variables update